Index page (#6526)
kyma-project/test-infra: README.md

  ├── .github                     # Pull request and issue templates             
  ├── development                 # Scripts used for the development of the "test-infra" repository
  ├── docs                        # Documentation for the test infrastructure, such as Prow installation guides

>   │    └── index.md               # Index page for test-infra documantation

Reviewer: ```suggestion
  │    └── index.md               # Index page for test-infra documentation
```

  └── prow                        # Installation scripts for Prow on the production cluster    

```

> See also [index page](https://github.com/kyma-project/test-infra/blob/main/docs/index.md) for test-infra documantation.

Reviewer: ```suggestion
See also [index page](/docs/index.md) for test-infra documentation.
```

> See also [index page](https://github.com/kyma-project/test-infra/blob/main/docs/index.md) for test-infra documantation.

Reviewer 2: ```suggestion
See also the [index page](https://github.com/kyma-project/test-infra/blob/main/docs/index.md) for test-infra documentation.
```
